CHANGELOG.md

```diff
@@ -59,6 +59,9 @@ Replaced a magic number used by the clock ([#4714](https://github.com/supercolli
 supernova now has more deterministic ordering of OSC messages in asynchronous requests ([#4460](https://github.com/supercollider/supercollider/pull/4460))
 
 ### scsynth and supernova: Fixed
+
+Fixed an issue with scsynth and supernova preproceeor directives ([#4784](https://github.com/supercollider/supercollider/pull/4784))
+
```

Comment:
Typo with `preprocessor`.

Might be good to group this with 4504 below, as it fixes something introduced in that PR?

Reply:
Fixed! Thanks